jobs.py

```diff
@@ -446,6 +446,19 @@
         "nb_pools": 2,
         "params": {},
         "paths": ["tests/misc/test_pool.py"],
+    },
+    "limit-tests": {
+        "description": "Tests verifying we can hit our supported limits",
+        "requirements": [
+            "1 XCP-ng host >= 8.2"
+        ],
+        "nb_pools": 1,
+        "params": {
+            # The test does not work on Alpine because of how it handles
+            # multiple interfaces on the same network, so use Debian instead
+            "--vm": "single/debian_uefi_vm",
+        },
+        "paths": ["tests/limits/test_vif_limit.py"],
```

Comment:
Why not just `tests/limits`? We can imagine other limit tests (vCPU, RAM, etc.)

Reply:
Changed to `tests/limits`